src/dodal/devices/fast_grid_scan.py

```diff
@@ -155,19 +170,16 @@ def grid_position_to_motor_position(self, grid_position: ndarray) -> ndarray:
 
 class GridScanParams(GridScanParamsCommon):
     """
-    Holder class for the parameters of a grid scan in a similar
-    layout to EPICS. These params are used for the zebra-triggered
-    fast grid scan
-
-    Motion program will do a grid in x-y then rotate omega +90 and perform
-    a grid in x-z.
-
-    The grid specified is where data is taken e.g. it can be assumed the first frame is
-    at x_start, y1_start, z1_start and subsequent frames are N*step_size away.
+    Params for standard Zebra FGS, which includes the dwell time
     """
 
     dwell_time_ms: float = 10
 
+    def get_param_positions(self):
```

Comment:
Not sure if this way of adding to the dictionary is any cleaner than the previous way of handling this